<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWin::KCMRulesList</class>
 <widget class="QWidget" name="KWin::KCMRulesList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="14">
    <widget class="QListWidget" name="rules_listbox"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="new_button">
     <property name="text">
      <string>&amp;New...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="modify_button">
     <property name="text">
      <string>&amp;Modify...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="delete_button">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="moveup_button">
     <property name="text">
      <string>Move &amp;Up</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="movedown_button">
     <property name="text">
      <string>Move &amp;Down</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>294</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="import_button">
     <property name="text">
      <string>&amp;Import</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="export_button">
     <property name="text">
      <string>&amp;Export</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
